x-pack/plugins/security_solution/server/lib/telemetry/event_based/events.ts

```diff
@@ -31,6 +32,12 @@ export const RISK_SCORE_EXECUTION_SUCCESS_EVENT: EventTypeOpts<{
         description: `Configured interval for the current risk scoring task`,
       },
     },
+    alertSampleSizePerShard: {
```

Comment:
I have added this to telemetry does anything need to be modified to get the telemetry cluster to retrieve this?

Reply:
I think that is enough. You can check if the telemetry is properly ingested on the staging env.

To do that you need to enable telemetry in `kibana.yml` `telemetry.optIn: true` and check https://telemetry-v2-staging.elastic.dev/s/securitysolution/app/dashboards.

Reply:
done